README.md

```diff
@@ -17,7 +17,7 @@ Latest:
 
 Add the following to your `project/plugins.sbt` or `~/.sbt/0.13/plugins/plugins.sbt` file:
 
-    addSbtPlugin("com.typesafe.sbt" % "sbt-git" % "1.0.0")
+    addSbtPlugin("com.github.sbt" % "sbt-git" % "1.0.0")
```

Comment:
I'd suggest just putting `<version>` in here, since otherwise it's likely to perpetually fall out of date, and because the GitHub web UI does a good job these days of foregrounding what the latest version of something is

Reply:
Done.